packages/perseus-editor/src/components/coordinate-pair-input.tsx

```diff
@@ -6,16 +6,22 @@ import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
-import type {LockedPointType} from "@khanacademy/perseus";
+import type {LockedCircleType, LockedPointType} from "@khanacademy/perseus";
 
 type Props = {
     coord: [number, number];
     error?: boolean;
-    onChangeProps: (newProps: Partial<LockedPointType>) => void;
+    // Most of the time, the prop to updated when this coordinate pair input
+    // is updated is is `coord`. For circles, it's `center`.
+    // This prop allows you to specify which prop to update.
+    changedProp?: "coord" | "center";
+    onChangeProps: (
+        newProps: Partial<LockedPointType> | Partial<LockedCircleType>,
```

Comment:
I think this should be `onChange: (coord: Coord) => void` and the parent should be the one to map the coordinate to an object, either `coord` or `center`.